### ISSUE TITLE
a rare new dragon

### ISSUE THREAD

**Issue by mypasswordissocool** [AlexModGuy/Ice-and-Fire-Suggestions#1290]
i think there should be a rare dragon type that spawns ONLY in the end highlands, only one of them can spawn in each highlands biome, and its a 10% chance one will even spawn, it would be called either the dark dragon or the void dragon, i will list some things that it could have and if you like it you can use it :D

Its breath attack will be deadly and blinding, when hit with it you will get blindness and wither (both last about 5 seconds after the breath stops)

its blood is black, and when put on a weapon it will do bonus damage to thunder dragons and other shadow dragons, (when attacking an enemy they will be give blindness and wither)

the scales of void dragons are very special and are able to lessen the affects of potions (as in a 30 second lasting potions affect will only last 15 seconds, so when you wear the armor you will have negated potion affects whether good or bad, this is also the case for the dragons them selves)

and finally void steel would give immunity to wither and blindness with the usual bonuses as well

__________________________________________________________________________________________________________________________________________
BONUS

when the breath hits the ground it will leave a dark mush covering blocks, this will cause blindness and wither for around 5 seconds, and the blocks that this breath create are dark like the night sky and very fragile like dirt

these creatures hatch from the darkness so to hatch them for yourself, you have 2 ways, the first way is to wait till night and place is under the stars, it can be in the end if you like, or the second way is to put it in a very low light level area (the first option is faster and hatches it 20 seconds faster)

These creatures are deaf due to the silence of the end and no need to worry about anything actually strong hurting them, this means that to wake them up you would have to hit them or bump into them, but this is also a weakness to you for the dragon whistle doesnt work on them, so be prepared to chase them if you are too slow to kill them before they take off.

i hope you enjoyed my idea :D

**Comment by Aceplante** [comment]
Absolutely! Perhaps the "ender dragon" that we know, is actually just one of many "void dragons" inhabiting the end...

Would go well with: https://github.com/AlexModGuy/Ice-and-Fire-Suggestions/issues/1315 (dragon behavior for ender dragon) in my opinion
